### DISCUSSION TITLE
Switching from Appc CLI to Ti CLI

### DISCUSSION THREAD

**Discussion by tyroneBaba** [appcelerator/titanium_mobile#12941] [Q&A] [answered]
How do you switch from using appc to ti cli?

**Comment by m1ga** [comment, marked as answer]
https://github.com/m1ga/from_zero_to_app/blob/master/appc_to_ti.md

**Reply by tyroneBaba** [reply]
Tyrone Babakhani 
thanks! trying the steps now

Michael Gangolf
if you are stuck somewhere or anything is missign give a quick feedback and I'll add it to the page

Tyrone Babakhani
stuck somewhere now actually :sweat_smile: the play/build button on vs code seems to still use appc, how do i change that?

Michael Gangolf
I only have Atom installed but VSCode should be the same. There is this option:
Screenshot_20210705_202326.png 
Screenshot_20210705_202326.png

Tyrone Babakhani
awesome! everything works now. thank you!

Some side notes: In addition to the steps above, I had to go into VSCode extension then near the bottom click: **Use Ti**. Then the plugin prompted me for a few updates to node and some other things. Some failed, so I closed/reopened VSCode. Ran the updates again, then it succeeded. "Fake" GUID was in place and removed all the "appc-" nodes at the bottom of our tiapp.xml. works like a charm


Thanks @m1ga !!
